src/utility/setLocalStorage.tsx

```diff
@@ -0,0 +1,30 @@
+import { Settings } from '../settings/settingsConfig';
+
+class LocalStorage {
+  isGeneratedMeals: boolean;
+
+  constructor(isGeneratedMeals: boolean) {
+    this.isGeneratedMeals = isGeneratedMeals;
+  }
+  saveSettings(storage: Settings) {
+    const jsonSettingsScreen = JSON.stringify(storage);
+    localStorage.setItem('settings', jsonSettingsScreen);
+  }
+  setEndTheDay(endTheDay: boolean) {
+    localStorage.setItem('isEndTheDay', endTheDay.toString());
+    return endTheDay;
+  }
+  getEndTheDay() {
+    return localStorage.getItem('isEndTheDay');
+  }
+  getIsGeneratedMeals() {
+    return localStorage.getItem('isGeneratedMeals');
+  }
+  setIsGeneratedMeals(isGenerated: boolean) {
+    this.isGeneratedMeals = isGenerated;
+    localStorage.setItem('isGeneratedMeals', this.isGeneratedMeals.toString());
```

Comment:
I think you can do without .toString, because local storage will do auto conversion

Reply:
if i do without .toString i get an error